### ISSUE TITLE
NVDA does not announce the banner role for a header tag in Chromium based browsers. 

### ISSUE THREAD

**Issue by rperez030** [nvaccess/nvda#12987]
### Steps to reproduce:

1. Open the attached 
[Test-page.txt](https://github.com/nvaccess/nvda/files/7414744/Test-page.txt)
example on the new Edge or chrome
2. Navigate between regions using the quick navigation key "D"

### Actual behavior:
For the banner section, nvda reads "landmark    This is a banner."
### Expected behavior:
NVDA should read "banner landmark    This is a banner."
#### NVDA installed/portable/running from source:
Installed copy.
#### NVDA version:
2021.2
#### Windows version:
Windows 11 version 21H2, build 22000.282
#### Name and version of other software in use when reproducing the issue:
Microsoft Edge version 95.0.1020.30
Google chrome 95.0.4638.54
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
[Test-page.txt](https://github.com/nvaccess/nvda/files/7414768/Test-page.txt)

**Comment by Adriani90** [comment]
I have UIA in edge and Chromium set to "only when suitable" which is the default behavior, and I cannot reproduce this issue with NVDA 2024.4 Beta on Chromium canary 131. Closing as works for me. If you are still having this issue, please comment and we can reopen.
